### ISSUE TITLE
Add support for new web-inspired a11y props

### ISSUE THREAD

**Issue by leggomuhgreggo** [FormidableLabs/eslint-plugin-react-native-a11y#152]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

#### Problem

The new web-inspired RN a11y props are not recognized by `eslint-plugin-react-native-a11y` rules.

Causes duplicate props to be added, with `--fix`

```
  25:5  error  Missing a11y props. Expected one of: accessibilityRole OR BOTH accessibilityLabel + accessibilityHint OR BOTH accessibilityActions + onAccessibilityAction  react-native-a11y/has-valid-accessibility-descriptors
```

![image](https://github.com/FormidableLabs/eslint-plugin-react-native-a11y/assets/2213636/e445104d-20bc-4ec2-8c44-f85f61b01040)

#### Context

React Native 0.71 introduced "Web-inspired props for accessibility, styles, and events" [[post](https://reactnative.dev/blog/2023/01/12/version-071#web-inspired-props-for-accessibility-styles-and-events)] 

> #### Accessibility
> We introduced ARIA props as alias to existing React Native accessibility props.
>
> These props now exist on all core components of React Native:aria-label, aria-labelledby,aria-modal, id, aria-busy, aria-checked, aria-disabled, aria-expanded, aria-selected, aria-valuemax, aria-valuemin, aria-valuenow, and aria-valuetext.
>
> We also introduced equivalent web behavior for: aria-hidden, aria-live, role, and tabIndex.
>
> See this [issue](https://github.com/facebook/react-native/issues/34424) for more details.

Here's some info on the updates in `react-native-web`:
https://github.com/necolas/react-native-web/issues/1112
https://github.com/necolas/react-native-web/releases/tag/0.19.0

**Comment by carbonrobot** [comment]
Duplicate of #151
